.travis.yml

```diff
@@ -9,15 +9,16 @@ branches:
   - develop
 
 env:
-- RUN_BACKEND_TESTS=true RUN_FRONTEND_TESTS=false REPORT_BACKEND_COVERAGE=false
-- RUN_BACKEND_TESTS=false RUN_FRONTEND_TESTS=true REPORT_BACKEND_COVERAGE=false
-- RUN_BACKEND_TESTS=true RUN_FRONTEND_TESTS=false REPORT_BACKEND_COVERAGE=true
+- RUN_BACKEND_TESTS=true RUN_FRONTEND_TESTS=false RUN_LINT=false REPORT_BACKEND_COVERAGE=false
```

Comment:
Can you reorder these so that:

- lint goes first
- frontend goes second
- backend without coverage goes third
- backend with coverage goes last?

The idea is to make failures apparent as early as possible. Actually, the lint probably takes longer than the frontend tests, but I figured we might as well try to keep the tests together.